concepts/05 Widgets/RadioGroup/00 Overview.md

```diff
@@ -17,10 +20,87 @@ The following code adds a simple **RadioGroup** to your page. Here, the [value](
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-radio-group
+        [data-source]="dataSource"
```

Comment:
In Angular, attributes use camelCase notation.

Reply:
Oh, sorry.. fixed